src/pages/Main.js

```diff
@@ -52,7 +52,6 @@ export default hot(module)(class Main extends Component {
         window.gapi.signin2.render('signInButton', {
           scope: [
             'profile', 'email', 'openid',
-            'https://www.googleapis.com/auth/devstorage.full_control',
```

Comment:
I really like the precision here, but I believe you can also remove the compute scope—I'm not sure we ever needed that. It's worth a quick test to make sure the FireCloud calls still work, but if so, drop it.